src/main/java/org/verapdf/pd/encryption/PDEncryption.java

```diff
@@ -0,0 +1,139 @@
+package org.verapdf.pd.encryption;
+
+import org.verapdf.as.ASAtom;
+import org.verapdf.cos.COSObjType;
+import org.verapdf.cos.COSObject;
+import org.verapdf.cos.COSString;
+import org.verapdf.pd.PDObject;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Represents encryption dictionary on PD level.
+ *
+ * @author Sergey Shemyakov
+ */
+public class PDEncryption extends PDObject {
```

Comment:
contains methods with similar logic. better to move that logic into separate method